udata_hydra/crawl.py

```diff
@@ -175,17 +202,11 @@ async def check_url(row, session, sleep=0, method="head"):
         )
         return STATUS_ERROR
 
-    should_backoff, nb_req = await is_backoff(domain)
+    should_backoff, reason = await is_backoff(domain)
     if should_backoff:
-        log.info(f"backoff {domain} ({nb_req})")
-        context.monitor().add_backoff(domain, nb_req)
-        # TODO: maybe just skip this url, it should come back in the next batch anyway
-        # but won't it accumulate too many backoffs in the end? is this a problem?
```

Comment:
You had some interesting concerns here, are they not relevant anymore?

Reply:
We'll see how it goes in real life but:
- it was an elegant way to handle the `x-rate-limit`, it wouldn't have worked with the previous pattern
- we're now working with a [smaller batch by default](https://github.com/etalab/udata-hydra/commit/5d2fd96a857b65a0e4d3cb952ecdf8f97ba357de) which makes me more confortable with this